<launch>
    <node pkg="slam2utm" type="slam2utm" name="slam2utm_converter" output="screen">
    
    
        <param name="bias_translation_x" value="1.0" />
        <param name="bias_translation_y" value="2.0" />
        <param name="bias_translation_z" value="3.0" />
        <param name="bias_twist_roll" value="0.1" />
        <param name="bias_twist_pitch" value="0.2" />
        <param name="bias_twist_yaw" value="0.3" />
    </node>
</launch>